plugins/healthcheck/healthz.go

```diff
@@ -60,8 +61,11 @@ func (i *healthcheckInstaller) Install(builder service.Builder, cfg *nirvana.Con
 			Consumes: []string{definition.MIMEAll},
 			Produces: []string{definition.MIMEAll},
 			Definitions: []definition.Definition{{
-				Method:   definition.Get,
-				Results:  []definition.Result{definition.ErrorResult()},
+				Method:  definition.Get,
+				Results: []definition.Result{definition.ErrorResult()},
+				Parameters: []definition.Parameter{
+					definition.QueryParameterFor("type", "the type of health check"),
```

Comment:
Why not use path like `/healthz/liveness`

Reply:
Fully fixed path or regexp path?

I believe that not everyone needs different types of checks. If you don't need that, just ignore the type parameter and use `/healthz`. In some other cases, we may need more types of checks.

Reply:
ping, any response? We need this feature to distinguish between different types of health checks (in our case, `liveness` and `readiness`)

Reply:
This is my point and not a problem. I think you need to solve the problem of `backward compatibility`.

Reply:
So how to support different type of health check without break anything?

Reply:
Add a new Configurer maker `func CheckerWithType(checker HealthCheckerWithType) nirvana.Configurer`. Then make `func Checker(checker HealthChecker) nirvana.Configurer` as an alternate.

Reply:
I made some changes as you request, PTAL